### DISCUSSION TITLE
positive ionization mode vs negative ionization mode

### DISCUSSION THREAD

**Discussion by tobigithub** [qcxms/QCxMS#40] [General]
Hi,
so I wonder how do we set **positive** ionization mode vs **negative** ionization mode in the qcxms.in for **CID mode**?

Is it automatically detected? Which would be nice, or do we set a specific keyword. The docs seem to only mention
positive ionization mode for CID (https://xtb-docs.readthedocs.io/en/latest/qcxms_doc/qcxms_ex.html?highlight=negative#dea-negative-ion-electron-ionization).

![image](https://user-images.githubusercontent.com/2569565/135174307-0250a822-7d2b-4735-bbca-3f13bce6f65e.png)

Tobias

**Comment by JayTheDog** [comment]
No, this is not yet implemented/fully tested. Settings have to be checked and the level of theory carefully considered. Might cost much more than pos. Ion mode because of larger basisfunctions that are needed to describe the negative charge accordingly.

**Comment by tobigithub** [comment]
I agree, it needs to be tested, but QCXMS (V5.1.2) works fine for small examples in negative ionization mode. Here 2-Mercaptoethanol in negative ionization CID mode (NCID), with 10V simulated and matching a triple quadrupole QQQ 18V spectrum. The scores are OK. Conformers play a role and of course protomers. I did not have any other instruments data from QTOFs or Orbitraps available, which would be nicer, because the fragments are accurate masses.

I think older semiempiricals should be fine in negative ionization mode, GFN-xTB2 should be also fine. There are noticeable differences for larger molecules (not shown here) when using electron affinity (EA) vs the ionization potential (IP) during the fragment calculation. IP is commonly used for positive mode, EA recommended for negative ion fragmentation regimes. I think the EA calculation creates many more smaller fragments, hence product ions can shift by 1-2 Dalton. So if IP or EA is more accurate during fragment and charge calculations has to be investigated for a larger number of molecules. But its nice that it conceptually works.

<p align="center">
<img align="center"  src="https://user-images.githubusercontent.com/2569565/138181842-bb46048c-11c6-4594-b03b-f3a066c603fc.png">
</p>

For this case the sulfur atom contained the negative charge and the sulfur atom was deprotonated. The deprotomers were calculated with CREST. The input settings were these (qcxms.in):
````
ncid
xtb2
elab 10
lchamb 0.25
fullauto
````
and the coordinate input in tmol format (coord):
````
$coord
    0.09887687797609     -0.49508410822139     -2.48202369602119      s
   -0.25701539167642     -1.59085379645756      2.77171466394524      o
   -0.40660619941421      1.92593253696480     -0.11752069596116      c
    0.47294125047760      0.92834749262411      2.45738170457888      c
    0.62763121865716      3.65400615131705     -0.61102298493010      h
   -2.43280722530313      2.37110542553999     -0.04787000438228      h
   -0.33870314375224      2.04837595700937      4.01093382326736      h
    2.55453643167356      1.08208275434676      2.56767444906756      h
   -0.25822768178814     -2.18813454579703      0.96853219212742      h
$end
````

**Comment by s645320041** [comment]
Hi,

I notice there is a 'ncid' in _qcxms.in_ and wonder if it's a new function in QCxMS but just not mentioned in docs?
(https://xtb-docs.readthedocs.io/en/latest/qcxms_doc/qcxms_run.html)

**Reply by JayTheDog** [reply]
Hi,
this is a something that will be available in the next release. This runmode has accidentally slipped into the latest release, but was not yet finished nor tested at that time.

**Comment by JayTheDog** [comment]
The negative Ion mode in QCxMS is available since the update to Version 5.2.0. 
Use that version with the keyword `charge <integer>` in the _qcxms.in_ file to obtain negatively charged CID mass spectra.
